accounts/accounts-build-ami.yml

```diff
@@ -0,0 +1,57 @@
+---
+resources:
+  - name: accounts-git
+    type: git
+    source:
+      uri: git@github.com:openstax/accounts.git
+      private_key: ((github-private-key))
+      branch: new_deployment
+
+  - name: concourse-pipelines
+    type: git
+    source:
+      uri: https://github.com/openstax/concourse-pipelines.git
+      branch: accounts-new-deploy-automation
+jobs:
+  - name: build-accounts-ami
+    plan:
+      - get: accounts-git
+        trigger: true
+      - get: concourse-pipelines
+
+      - task: get-accounts-sha
+        config:
+          inputs:
+            - name: accounts-git
+          outputs: 
+            - name: accounts-sha
+          platform: linux
+          image_resource:
+            type: docker-image
+            source: 
+              repository: scb6/alpine-git
+
+          run:
+            path: sh
+            args: ["-c", "git -C ./accounts-git rev-parse HEAD > ./accounts-sha/sha"]
```

Comment:
the concourse git resource will have created a file in `accounts-git/.git/short_ref`, so this task might not be necessary

https://github.com/concourse/git-resource#additional-files-populated